### PR TITLE
[Java] Code Samples updated and minor doc fixes

### PR DESCRIPTION
**Pull request by MrRahulR**
### Description
Some of the required examples for the Selenium Java methods are added in the document.

### Motivation and Context
The change will add more context to the Selenium methods and give some better examples to use for developers.

### Types of changes
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/elements/information.en.md [hunk 1]
```diff
@@ -50,7 +54,7 @@ let result =  await driver.findElement(By.css("[name='btnK']")).isDisplayed();
 {{< /tabpane >}}
 
 {{< alert-code >}}
-for element displayedness
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/information.en.md]: I don't know what much difference between `element displayedness` and `isDisplayed`

Basically `element displayedness` is more generic and it is as it is according to W3C doc!

Could you please revert it!

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: website_and_docs/content/documentation/webdriver/elements/information.en.md (hunk 1 above)
Comment (MrRahulR) [reply website_and_docs/content/documentation/webdriver/elements/information.en.md]: @harsha509 changes reverted to `element displayedness`

**Review by harsha509 — APPROVED** [review]
Thank you @MrRahulR !

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium Docs @MrRahulR  🎉 🎉 !

**Pull request merged**